grails-app/views/comment/_comment.gsp

```diff
@@ -0,0 +1,78 @@
+<div id="commentOutput" class="clearfix">
+    <h3>Comments (<b data-bind="text: total"></b>)</h3>
+    <div id="postComment" class="comment clearfix comment-post">
+        <div class="comment-header">
+        </div>
+
+        <div class="comment-body">
+            <textarea class="boxsizingBorder" data-bind="value: newComment().text" placeholder="Write your comment here."></textarea>
+        </div>
+
+        <div class="comment-footer pull-right">
+            <div class="btn btn-primary btn-small" data-bind="click: create">post</div>
+        </div>
+    </div>
+    <hr>
+    <div id="commentDisplay">
+        <div id="commentTools">
+            <span>
+                <select
+                        data-bind="options: sortOptions, optionsText: 'text', value: selectedSort, event:{change: list}"></select>
+            </span>
+        </div>
+
+        <div class="comment-list" data-bind="template:{name:'template-comment',foreach: comments}">
+        </div>
+    </div>
+    <div data-bind="visible: showLoadMore" class="row-fluid">
+        <div class="span12">
+            <div class="btn span12" data-bind="click: more"><span class="h6">load more</span></div>
+        </div>
+    </div>
+</div>
+<script type="text/html" id="template-comment">
+    <div class="comment-body">
+        <div class="media" >
+            <div class="pull-left" href="#" style="width: 16px; height: 16px" data-bind="css: { hide: !$data.parent()}"></div>
```

Comment:
Attribute href is not valid on a div....

Don't use inline styles - put it in the .css file.